hbase_master/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
Please update the csv to list the metrics collected (see more here: http://docs.datadoghq.com/guides/integration_sdk/#metadatacsv)